cmd/swarm/access_test.go

```diff
@@ -52,11 +52,12 @@ func TestACT(t *testing.T) {
 		t.Skip()
 	}
 
-	initCluster(t)
+	cluster := newTestCluster(t, clusterSize)
```

Comment:
why are you reverting this change? @nonsense introduced `initCluster` to save time and resource on travis builds. please elaborate why this is needed

Reply:
@frncmx please just confirm that the amount of time spend in these tests is not increasing with respect to `master`. Spinning up a new cluster is slow and we don't want to add 10sec. per test for that (not saying that you are doing that, just make sure this is the case please :)

Reply:
@justelad with the global (singleton) cluster, I had some problems answering the following questions:
- Who should be responsible for cleaning up the cluster? 
- Why do we keep the cluster running (consuming ~1.5GB of memory) for the whole CI run while not using it most of the time?
- Is it a good idea for tests to share state across files?

Note: I did not completely revert. I kept the sub tests from the original file. So we still have speed up.

Reply:
@justelad  Oh, also note, I made a mistake. Some cleanup and a change to another function stayed in my stash. I'll push that promptly.

Reply:
@nonsense we use `initCluster()` at 2 places. I plan to eliminate both. So we start the cluster 2 times instead of 1. But at the same time I made an other commit (adding parallelization) that exactly provides a 50% speedup for 3 nodes. Stay tuned; I managed NOT to push all my changes by a mistake.

Reply:
> Who should be responsible for cleaning up the cluster?

No one. We simply don't clean it up. Why? because the data that is pumped into it is random anyway, so bringing up nodes to pump random data then take them down to do the same again is not effective. Felix suggested this solution and I think he is also right.

> Why do we keep the cluster running (consuming ~1.5GB of memory) for the whole CI run while not using it most of the time?
Is it a good idea for tests to share state across files?

This solution was born out of pragmatism, not by being pedantic. Ethereum CI runs used to be more than half an hour long, each, meaning when you push something you'd end up waiting a very long time to see if you broke something or not which is very frustrating. That was mostly caused because of us. Reusing the cluster in `cmd/swarm` took off 1 minute from every CI run. If you'd check the file history and see [this PR](https://github.com/ethereum/go-ethereum/pull/18129) you'd see this exactly. And BTW benchmarking the diff on your machine doesn't count - benchmark it on Travis and see if you add time or not (the specs on your machine are light years from the CI instances we get on ethereum and ethersphere. The latter being much less powerful AFAIK)

Reply:
@frncmx: as @justelad says, we keep the cluster running, because it is slow to start it and we don't want our test suite to be slow.

Pushing chunks that are random from one run to the next should have no effect on test runs - basically what you are noticing is not done by mistake, but on purpose.

Reply:
@frncmx I am voting no for your suggestion, because this will keep adding time to our test suite. I would be happy to start the cluster N times instead of 1, if we get the chance to move to a faster CI.

Reply:
@frncmx `initCluster` might be visible at two places in the code, but it is shared in a table-driven test at `TestACT` among 4 tests, with only one occurrence in the code.